src/providers/ThemeProvider.tsx

```diff
@@ -3,8 +3,12 @@ import { useJuiceTheme } from 'hooks/JuiceTheme'
 
 export const ThemeProvider: React.FC = ({ children }) => {
   const juiceTheme = useJuiceTheme()
+  const isDarkMode = juiceTheme.isDarkMode
 
   return (
-    <ThemeContext.Provider value={juiceTheme}>{children}</ThemeContext.Provider>
+    <ThemeContext.Provider value={juiceTheme}>
+      {/* Set the dark mode flag if dark mode is enabled */}
+      <div className={isDarkMode ? 'dark' : undefined}>{children}</div>
```

Comment:
**suggestion** we could save a `div` here by adding the class to the CoreAppWrapper instead:

https://github.com/jbx-protocol/juice-interface/blob/b42064d3f0ce25420453b81270b49733b0bbd2aa/src/components/common/CoreAppWrapper/CoreAppWrapper.tsx#L56

Reply:
I chose ThemeProvider to be intentful as it is related to colors. Adding to top of Layout saves a div, but isn't as meaningful imo.

I am hoping we shift off class to media queries asap